src/client/Microsoft.Identity.Client/Http/DefaultRetryPolicy.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.Http
```

Comment:
Is it worth moving all these Retry clasess to their own "Retry" folder?

Reply:
Should the "Retry" folder be inside of the "Http" folder?

Reply:
done: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5300